app/presenters/submission_presenter.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class SubmissionPresenter < SimpleDelegator
```

Comment:
Is this presenter necessary for a single attribute? I realise this is a pattern we're using throughout the application, so might make sense for consistency.

Reply:
I've added it as future proofing because we'll probably add more things but... maybe overkill for now??

Reply:
Making an executive decision to keep it ;)